### PR TITLE
add Input UI

### PR DESCRIPTION
**Pull request by Habibayv28**
This branch was created to add Input UI design to the project .
I am requesting @Kachulio1 @daveclinton @asiimwe-a @gitahievans  for a review before merging to the Dev Branch

### CODE REVIEW

File: src/components/InputField/input.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
```
Comment (daveclinton) [inline src/components/InputField/input.tsx]: Remove elint disable comment

File: src/App.tsx [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+import Searchbar from "./components/InputField/input";
 
 const App: React.FC = () => {
   return (
-    <Box backgroundColor="red">
-      <Text color="orange" fontSize="100px">
-        Hello World
-      </Text>
+    <Box backgroundColor="white">
+      <Searchbar setSearch={function (value: React.SetStateAction<string | undefined>): void {
```
Comment (daveclinton) [inline src/App.tsx:8]: What is the purpose of setSearch?

**Review by daveclinton — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 2 above)
Comment (Habibayv28) [reply src/App.tsx:8]: Detects if the user is pressing the input to start a search.

File: src/components/InputField/input.tsx (hunk 1 above)
Comment (Habibayv28) [reply src/components/InputField/input.tsx]: this has been resolved

**Review by daveclinton — APPROVED** [review]

**Pull request merged**